nautobot/extras/management/commands/runjob.py

```diff
@@ -53,6 +55,13 @@ def handle(self, *args, **options):
             request.id = uuid.uuid4()
             request.user = user
 
+        data = {}
+        try:
+            if options.get("data"):
+                data = json.loads(options["data"])
+        except json.decoder.JSONDecodeError as error:
+            raise CommandError(f"Original Error: {str(error)}\n\n invalid json data.\n\n")
```

Comment:
I don't think all those newlines are needed?

```suggestion
            raise CommandError(f"Invalid JSON data:\n{str(error)}")
```